info.json

```diff
@@ -217,7 +217,7 @@
         ],
         "recordSets": [
             {
-                "name": "Announcements",
+                "name": null,
```

Comment:
update the value of key "name" to "Announcements"

Reply:
done